modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -162,7 +162,7 @@ export default function EditNotificationTemplate({
 				fromName: {
 					[defaultLanguageId]: '',
 				},
-				singleRecipient: false,
+				singleRecipient: !Liferay.FeatureFlags['LPS-187854'],
```

Comment:
@seltong I don't think we should add this property if the FF is disabled. Besides that we would add it with the value `true` which is not expected. You can add the following lines after the initialization of`recipientInitialValue`:
```
if(Liferay.FeatureFlags['LPS-187854']) {
	recipientInitialValue[0] = {...recipientInitialValue[0], singleRecipient: false};
}
```

Reply:
Oh, I thought the current behavior was the same as when we have the singleRecipient set to false. I'll check the story again.

Reply:
We've discussed and agreed that the current behavior is the same as singleRecipient set to true. My bad!